### DISCUSSION TITLE
GitHub Sponsors for autokey?

### DISCUSSION THREAD

**Discussion by tblancher** [autokey/autokey#980] [Q&A]
Today I was reminded that GitHub allows people to sponsor their favorite repositories and open source projects.  I just set up monthly contributions to two projects I use nearly every day.

I'd like to do the same for Autokey.  Is this something the project would like?

**Comment by josephj11** [comment]
Wow! Thank you so much! AutoKey has no formal organization, so the project has no way to accept funds directly.

There are at least two other options.

You can fund bounties through [Opire](https://github.com/autokey/autokey/wiki/Contributing#donations). That will require your attention in deciding when a goal has been completed, etc.

You can offer funds directly to one or more project members. This has never been done yet.
